compiler/rustc_target/src/target_features.rs

```diff
@@ -765,13 +765,13 @@ const LOONGARCH_FEATURES: &[(&str, StabilityUncomputed, ImpliedFeatures)] = &[
     // tidy-alphabetical-start
     ("d", unstable(sym::loongarch_target_feature), &["f"]),
     ("f", unstable(sym::loongarch_target_feature), &[]),
-    ("frecipe", unstable(sym::loongarch_target_feature), &[]),
-    ("lasx", unstable(sym::loongarch_target_feature), &["lsx"]),
-    ("lbt", unstable(sym::loongarch_target_feature), &[]),
-    ("lsx", unstable(sym::loongarch_target_feature), &["d"]),
-    ("lvz", unstable(sym::loongarch_target_feature), &[]),
-    ("relax", unstable(sym::loongarch_target_feature), &[]),
-    ("ual", unstable(sym::loongarch_target_feature), &[]),
+    ("frecipe", STABLE, &[]),
+    ("lasx", STABLE, &["lsx"]),
+    ("lbt", STABLE, &[]),
+    ("lsx", STABLE, &["d"]),
+    ("lvz", STABLE, &[]),
+    ("relax", STABLE, &[]),
+    ("ual", STABLE, &[]),
```

Comment:
- `relax` refers to linker relaxation support and may not be appropriate to expose as a target feature.
- `ual` refers to support for unaligned memory access, similar to `strict-align` on ARM targets. It's probably fine to stabilize as-is, but we may want to double-check with other targets if we want to have a consistent feature name between all of them (alternatively it's also fine to make the feature target-specific).

Reply:
Also to be clear, `ual` can only affect how unaligned accesses are codegen'd, and nothing else.  See [this recent discussion](https://rust-lang.zulipchat.com/#narrow/channel/242906-t-compiler.2Farm/topic/strict-align.20target.20feature.3F) on Zulip.

If we have target feature docs somewhere, they should clarify hat unaligned accesses are still UB except when performed via `{read,write}_unaligned`.

Reply:
I agree with skipping these two target features and stabilizing them at a more appropriate time.